### ISSUE TITLE
Feature Request  Android 360 Cam

### ISSUE THREAD

**Issue by stilicrafter** [mapillary/mapillary_issues#2431]
Hi,
please add 360 cam support for cameras in the Android app.

**Comment by vincentdephily** [comment]
Gah, my LG360 got delivered today, I bought it mainly for Mapillary, and now I learn it's not supported by Mapillary Android. No mention of this showstoper fact on https://www.mapillary.com/360.

Is the feature right around the corner, or months away, or... ?

**Comment by millenbop** [comment]
It does say that it only works on iOS but it could be clearer.

<img width="588" alt="screen shot 2017-01-31 at 13 38 26" src="https://cloud.githubusercontent.com/assets/12835674/22465166/a3445916-e7ba-11e6-937a-9dba144a2ba3.png">

@rsahlin is working on adding 360 support for Android right now but I don't think we can give you an estimate of when it's ready.

**Comment by peterneubauer** [comment]
Yup,
this is in progress, we are starting on it in Android now, but need some features before - like recording the GPX track so we can sync the images from the LG before upload.

**Comment by vincentdephily** [comment]
Thanks for the update. Looking forward to Mapillary support, as LG's app sometimes lets the GPS turn itself off, making lots of photos unusable for mapping purposes.

**Comment by frafra** [comment]
It would be very nice to be able to use it in Android :) Any progress/beta version we could test?

**Comment by ianatkin80** [comment]
I would like to add my support to this issue, is there an ETA for implementation?

**Comment by peterneubauer** [comment]
Yes, this is the next thing we are working on. Also, the GPX file support is now in, so in principle, you can record a sequence, and then drop into thatt sequence any external (time-synched) image files, even LG 360, and upload them like normal (you can delete the original ones if you like). Try it out and let us know!

**Comment by utack** [comment]
Hi. Is there any progress on this?

**Comment by ovivoda** [comment]
_mapillary_issues is no longer an active repository._

_Let’s continue discussing features in our forum! We will import this issue as topic to the <a href="http://forum.mapillary.com ">Mapillary Forum</a> in the coming weeks. We’ll notify here once it’s been done. See you there!_
